diff --git a/configure.ac b/configure.ac
index a4645bd..f0b4ab2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -35,10 +35,8 @@ m4_define([gimp_data_version], [2.0])
 m4_define([gimp_sysconf_version], [2.0])
 m4_define([gimp_user_version], [2.9])
 
-m4_define([gimp_unstable],
-          m4_if(m4_eval(gimp_minor_version % 2), [1], [yes], [no]))
-m4_define([gimp_stable],
-          m4_if(m4_eval(gimp_minor_version % 2), [0], [yes], [no]))
+m4_define([gimp_unstable],[no])
+m4_define([gimp_stable],[yes])
 
 m4_define([gimp_full_name], [GNU Image Manipulation Program])
 
